src/IconToggle/index.js

```diff
@@ -2,13 +2,36 @@ import React, { Component } from 'react'
 import { View, StyleSheet } from 'react-native'
 import { IconToggle } from '@protonapp/react-native-material-ui'
 
+
+
 export default class WrappedIconToggle extends Component {
   handlePress = async () => {
     let {
       clickActions,
       input: { value, onChange },
     } = this.props
-    await onChange(!value)
+    //await onChange(!value)
+
+    let {
+      activeActions,
+      input: { value, onChange },
+    } = this.props
+    await onChange(value == true)
```

Comment:
```suggestion
    await onChange(!value)
```
I'm pretty sure `value` reflects the current value of the prop, and `onChange` is how you change that value. As long as that is the case (you'd be able to test this by logging it out into the dev console), the toggle should flip `value` to be whatever it isn't currently.